.travis.yml

```diff
@@ -0,0 +1,9 @@
+before_install:
```

Comment:
you don't need the manual installation of yarn because travis ci will detect the `yarn.lock` file and install it for us